helm/prometheus-rules/templates/alerting-rules/kiam.rules.yml

```diff
@@ -27,4 +27,21 @@ spec:
         severity: page
         team: firecracker
         topic: kiam
+  - name: kiam
+    rules:
+    - alert: KiamSTSIssuingErrors
+      annotations:
+        description: '{{`Kiam pod {{ $labels.namespace}}/{{ $labels.pod_name }} on {{ $labels.cluster_id}}/{{ $labels.cluster }} has increased STS issuing errors.`}}'
+        opsrecipe: kiam_sts_issuing_errors_total/
+      expr: increase(kiam_sts_issuing_errors_total[10m]) > 0
```

Comment:
Maybe we could allow some errors? Like 5 or 10 dunno

Reply:
Makes sense 👍🏻